Automattic/wp-calypso
# Account Recovery: Add redux state tree and undocumented api endpoint wrapper

# Account Recovery: add api and redux state tree

This pull request adds a wrapper for a new undocumented api endpoint that performs a lookup for a users' account recovery options.

### How to test
1. Navigate to http://calypso.localhost:3000/account-recovery
2. Enter a username
3. Click "Get new password"
4. Notice that the `ACCOUNT_RECOVERY_LOOKUP_REQUEST` and `ACCOUNT_RECOVERY_LOOKUP_RECEIVE` actions were dispatched
